db/seeds.rb

```diff
@@ -47,3 +49,79 @@ class SeedAdminENVError < StandardError
 business_categories.each { |b_category| BusinessCategory.find_or_create_by(name: b_category) }
 BusinessCategory.find_or_create_by(name: 'Sociala tjänstehundar', description: 'Terapi-, vård- & skolhund dvs hundar som jobbar tillsammans med sin förare/ägare inom vård, skola och omsorg.')
 BusinessCategory.find_or_create_by(name: 'Civila tjänstehundar', description: 'Assistanshundar dvs hundar som jobbar åt sin ägare som service-, signal, diabetes, PH-hund mm')
+
+if Rails.env.development? || Rails.env.staging?
+  r = Random.new
+  USERS = 40
+  regions = Region.all.to_a # regions are created in a migration file
+  num_regions = regions.size
+
+  # Create users
+  users = []
+
+  USERS.times do
+    users << User.create(email: FFaker::InternetSE.free_email,
+                         password: 'password')
+  end
+
+  puts "Users created: #{User.all.count}"
+
+  # Create membership application for some users
+  # (two rounds - so some of the users have more than one application)
+
+  business_categories = BusinessCategory.all.to_a
+  num_cats = business_categories.size
+
+  applications = []
+
+  2.times do
+    r.rand(1..USERS).times do
+
+      company_number = r.rand(1000000000..9999999999).to_s
```

Comment:
Just checking:   Even though you're not validating when you save, this company number won't cause any validation problems later, right?
i.e. `swedish_organisationsnummer` will pass with this `company_number` if one of these are saved later